project/Dependencies.scala

```diff
@@ -3,16 +3,15 @@ import sbt._
 object Dependencies {
 
   val catsEffect = "org.typelevel" %% "cats-effect" % "2.0.0"
+  val http4sBlazeClient = "org.http4s" %% "http4s-blaze-client" % "0.20.11"
+  val http4sBlazeServer = "org.http4s" %% "http4s-blaze-server" % "0.20.11"
+  val http4sDsl = "org.http4s" %% "http4s-dsl" % "0.20.11"
```

Comment:
this is a good example, where having versions of packages in `Versions` is good ;)

Reply:
You are absolutely right but my point is that I can do it now once I actually need it, there is no need to introduce the `Versions` object when all the dependencies are alone. Anyway, I am not going to do it because of Scala Steward, I don't think it could handle, or do you?

Reply:
it works! :tada:

https://github.com/typelevel/frameless/pull/397/files

cc @augi

Reply:
OK. But I'll do just the ones that are duplicated, not all of them.